it/mail/chrome/messenger/preferences/advanced.dtd

```diff
@@ -48,12 +48,12 @@
 <!ENTITY enableAddonsUpdate.label       "Componenti aggiuntivi installati">
 <!ENTITY enableAddonsUpdate.accesskey   "C">
 <!ENTITY checkNow.label                 "Controlla ora">
-<!ENTITY whenUpdatesFound.label         "Quando viene trovato un aggiornamento per &brandShortName;:">
-<!ENTITY modeAskMe.label                "Chiedi che cosa fare">
+<!ENTITY whenUpdatesFound.label         "Quando è disponibile un aggiornamento per &brandShortName;:">
+<!ENTITY modeAskMe.label                "Chiedimi che cosa fare">
 <!ENTITY modeAskMe.accesskey            "e">
-<!ENTITY modeAutomatic.label            "Scarica ed installa automaticamente l'aggiornamento">
+<!ENTITY modeAutomatic.label            "Scarica e installa automaticamente l'aggiornamento">
 <!ENTITY modeAutomatic.accesskey        "S">
-<!ENTITY modeAutoAddonWarn.label        "Avvisa se verranno disattivati dei componenti aggiuntivi">
+<!ENTITY modeAutoAddonWarn.label        "Avvisa se verranno disattivati alcuni tra i miei componenti aggiuntivi">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY modeAutoAddonWarn.label        "Avvisa se verranno disattivati alcuni dei miei componenti aggiuntivi">